### DISCUSSION TITLE
Spaces in commad line options

### DISCUSSION THREAD

**Discussion by ossodue** [joncampbell123/dosbox-x#4418] [Q&A]
Hello,
I'd like, for example, to set a config option using "set parallel1=file dev:lpt1" in the command line option but it doesn't works because
the string " dev:lpt1" is ignored.
If I use "set parallel1="file dev:lpt1"" the paralle1 property is set to "file.

How can I resolve it? Is there a special escape character to use?

Thank you.

**Comment by maron2000** [comment]
How to quote an command line option is mentioned in the Wiki (https://github.com/joncampbell123/dosbox-x/wiki#dosbox-xs-configuration-file)
```
If there is at least one space in the config option (e.g. dpi aware in [dosbox] section) then you will need to put
the section name in front of it and quote the whole string, such as -set "dosbox dpi aware=true".
```

TL:DR
`dosbox-x -set "parallel1=file dev:lpt1"`
